test/intest7.F90

```diff
@@ -144,7 +144,7 @@ program intest7
   if ( ( nsub .ne. 402 ) .or. ( ibfms ( r8val ) .ne. 1 ) ) stop 18
 
   ! Test the error handling inside of VALX.
-  errstr_len = 1
+  errstr_len = 0
```

Comment:
What is going on here? How come you''re changing the errstr_len to 0?

Reply:
Good question!  When I was copying the errwrt subroutine and Share_errstr module over from one of my other test codes, I realized that errstr_len should be 0 instead of 1, because the logic in the errwrt subroutine fills the errstr array starting with byte errstr_len+1, rather than starting with byte errstr_len.  So it hadn't been causing any problems, but it also was never using the first byte of the array and was always filling it beginning with byte 2.  So I went ahead and fixed it in the other test codes as well.